_tech/electrical/electrical.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Electrical Overview
+---
+
+The Electrical system is consisted up of everything that gets controlled by the [Controls System](control-system)
```

Comment:
It seems like it would be a good idea to show 'related articles' here using tags. See the pneumatics section for example.